### ISSUE TITLE
Option for node-sass?

### ISSUE THREAD

**Issue by kevinSuttle** [yeoman/generator-angular#698]
I've been using it exclusively for some time now with no problems. I also personally don't use Compass, so I wouldn't miss it. I know there are those that do, though.

**Comment by hanfeisun** [comment]
:+1:

**Comment by subarroca** [comment]
:+1:

**Comment by schmitzdenis** [comment]
:+1:

**Comment by ardeearam** [comment]
:+1:

**Comment by chemicalbr** [comment]
+1

**Comment by AlJohri** [comment]
https://github.com/yeoman/generator-webapp/issues/484

:+1:

**Comment by jking90** [comment]
:+1:

**Comment by arthurvr** [comment]
These days generator-webapp and gulp-webapp don't even have the option anymore: https://github.com/yeoman/generator-webapp/commit/887d96bae2870d8b02fad8bcced2a1bb140a8d18, it's all libsass :)

**Comment by guy-mograbi-at-gigaspaces** [comment]
+1

**Comment by realityking** [comment]
This should probably not be done until sass/libsass#1115 is fixed, as it breaks Bootstrap 3.

**Comment by AlJohri** [comment]
or use any version prior to ```3.2.0-beta.5```

**Comment by eddiemonge** [comment]
https://github.com/yeoman/generator-angular/issues/819#issuecomment-100379175

**Comment by mininao** [comment]
:+1: This would be extremely useful !

**Comment by javarch** [comment]
Ich werde ab  16.09.2015 nicht im B�ro sein. Ich kehre zur�ck am
12.10.2015.
Ich werde Ihre Nachricht nach meiner R�ckkehr beantworten.

Die R�ckmeldung bezieht sich auf ein Mail mit folgendem Thema:
Re: [generator-angular] Option for node-sass? (#698)

____________________________________________________________________________________________
Gesendet (c) GRZ/RACON Linz 2015 Agent 'Abwesenheit'

Raiffeisen Software GmbH
Firmenbuchnummer: 86804d, Landesgericht Linz
DVR: 0572934, UID: ATU 36961105

Der Austausch von Nachrichten mit dem oben angef�hrten Absender via E-Mail
dient ausschlie�lich Informationszwecken. Rechtsgesch�ftliche Erkl�rungen
d�rfen �ber dieses Medium nicht ausgetauscht werden.
Correspondence with above mentioned sender via e-mail is only for
information purposes. This medium is not to be used for the exchange of
legally-binding communications.

**Comment by raphinesse** [comment]
I could file a PR that removes `compass` and uses `node-sass` instead. Would that be desirable or does it *have* to be an option?

**Comment by redfox05** [comment]
@raphinesse an option would be good for those who do/dont want to use it. I'm trying to go with the instruction on this site (http://ravelantunes.com/blog/angular-generator-libsass/), but does that mean I need to run the angular generator with the SASS + Compass option selected, and then remove it afterwards? Whats the best option? Basically just want to remove all dependancy on Ruby, and compass using Ruby, so will node-sass do what compass does. What about the image stuff? The link I referenced does not seem to address that part.

**Comment by redfox05** [comment]
seems something similar has already been done for generator-webapp (https://github.com/yeoman/generator-webapp/issues/302) could someone look to do the same for generator-angular?

**Comment by perpantha** [comment]
Yeah this would be extremely useful. Personally I don't use compass much and it just adds another dependency for my server guys when getting the projects up and running :)

**Comment by mbejda** [comment]
+1

**Comment by rafegoldberg** [comment]
Compass is old and tired so two 👍 for this option. Or you can follow the instructions in [this comment](https://github.com/yeoman/generator-angular/issues/819#issuecomment-100379175) for how to do it manually.
